### ISSUE TITLE
Feature request: mpris - configurable number of spaces between buttons

### ISSUE THREAD

**Issue by mxmilkb** [ultrabug/py3status#1737]
Even with no spaces in the config format string, it appears there are three spaces between the mpris buttons.

![rose_2019-03-07-02 54 56](https://user-images.githubusercontent.com/108225/53929082-7425b600-4084-11e9-8882-07a8135c54ea.png)

**Comment by lasers** [comment]
Quick solution.
* Use `py3status --include ~/.config/py3status/modules`.
* Place a copy of modified `mpris.py` with this diff.
* Adjust `min_width` to your likings. 
```diff
diff --git a/py3status/modules/mpris.py b/py3status/modules/mpris.py
index 8c2705c..cad64dc 100644
--- a/py3status/modules/mpris.py
+++ b/py3status/modules/mpris.py
@@ -303,7 +303,7 @@ class Py3status:
                 "color": color,
                 "full_text": control_state["icon"],
                 "index": button,
-                "min_width": 20,
+                "min_width": 10,
                 "align": "center",
             }
 
```

**Comment by mxmilkb** [comment]
Thanks!

**Comment by lasers** [comment]
Closed via #1738.
